src/pymovements/events/events.py

```diff
@@ -125,15 +125,15 @@ def schema(self) -> pl.type_aliases.SchemaDict:
     def __len__(self) -> int:
         return self.frame.__len__()
 
-    def __getitem__(self, *args, **kwargs) -> Any:
+    def __getitem__(self, *args: str, **kwargs: str) -> Any:
```

Comment:
the `__getitem__()` method from `polars.DataFrame` is overloaded with a range of different types:
https://github.com/pola-rs/polars/blob/master/py-polars/polars/dataframe/frame.py#L1467-L1657

`polars` is actually a bit confusing when used with this traditional bracket indexing.
`df['a']` will get you the column 'a' as a series, `df[1]` will give you the first row as a dataframe and
`df[1, 'a']` will give you the value of the specific cell.

maybe we should use the following signature to be a bit more  consistent with the polars interface?
```
    def __getitem__(
        self,
        item: str | int | np.ndarray[Any, Any] | tuple[int, int] | tuple[int, str]),
    ) -> Any:
```

 or the full deal by supporting slices as indices:

```
MultiRowSelector: TypeAlias = slice | range | list[int] | pl.Series
MultiColSelector: TypeAlias = slice | range | list[int] | list[str] | list[bool] | pl.Series

def __getitem__(
        self,
        item: (
            str
            | int
            | np.ndarray[Any, Any]
            | MultiColSelector
            | tuple[int, MultiColSelector]
            | tuple[MultiRowSelector, MultiColSelector]
            | tuple[MultiRowSelector, int]
            | tuple[MultiRowSelector, str]
            | tuple[int, int]
            | tuple[int, str]
        ),
    ) -> DataFrame | Series:
```

Reply:
done